src/main/java/utils/Listener.java

```diff
@@ -0,0 +1,38 @@
+package utils;
+
+import baseEntities.BaseTest;
+import io.qameta.allure.Attachment;
+import org.openqa.selenium.*;
+import org.openqa.selenium.support.events.WebDriverListener;
+import org.testng.ITestListener;
+import org.testng.ITestResult;
+
+public class Listener implements ITestListener, WebDriverListener {
```

Comment:
Name "Listener" doesn't provide any information about what is happening inside it